plot.py

```diff
@@ -603,6 +606,7 @@ def _update_plot(self):
                                                 norm=norm, orientation='horizontal')
         # colorbar - use specified units or default depending on
         # what has or has not been entered
+        #XXX this should be handel else where
         if self.units is None or self.units == '':
```

Comment:
self.Vradar.value.fields[self.field]['units'] should be the standard value for unit not None